Saves the username and password of the iTunes account into the keychain (#25)
interstateone/xcodes: README.md

You'll then be prompted to enter your Apple ID username and password. You can also provide these with the `XCODES_USERNAME` and `XCODES_PASSWORD` environment variables.

> After successful log in, xcodes will save your Apple ID password into the keychain for quick retrieval subsequent tries.

Reviewer: ```suggestion
After successful login, xcodes will save your Apple ID password into the keychain for quick retrieval on subsequent tries.
```
